chalicelib/app_utils.py

```diff
@@ -1,10 +1,15 @@
+import os
 from os.path import dirname
 from foursight_core.app_utils import AppUtils as AppUtils_from_core
 from .vars import FOURSIGHT_PREFIX, HOST
 
 
 class AppUtils(AppUtils_from_core):
 
+    es_host_from_foursight_core_apply_identity = os.environ.get("ES_HOST")
+    if es_host_from_foursight_core_apply_identity:
```

Comment:
What happens if this is not set? Should there be an error raised?

Reply:
Hm, yeah, probably should raise an error; will make that change.